website/docs: improve-rac-documents (#14414)
goauthentik/authentik: website/docs/add-secure-apps/providers/rac/how-to-rac.md

title: Create a Remote Access Control (RAC) provider
---

> ## Introduction

Reviewer: I'd leave this out... most of our docs start with a few words about the topic below, it is understood to be the Intro section.

> ## Introduction

Reviewer: ```suggestion
```

> The RAC provider is a highly flexible feature for accessing remote machines.

Reviewer: ```suggestion
The Remote Access Control (RAC) provider is a highly flexible feature for accessing remote machines.
```

> Fow more information about using the RAC provider, see the [Overview](./index.md) documentation. You can also view our video on YouTube for setting up RAC.

Reviewer: ```suggestion
For overview information, see the [Remote Access Control (RAC) Provider](./index.md) documentation. You can also view our video on YouTube for setting up RAC.
```

<iframe width="560" height="315" src="https://www.youtube.com/embed/9wahIBRV6Ts;start=22" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>

## Prerequisites

The RAC provider requires the deployment of the [RAC Outpost](../../outposts/index.mdx).

## Overview workflow to create an RAC provider

The typical workflow to create and configure a RAC provider is:

> 1. Create an application and provider.

Reviewer: so much better, thanks!

            - **Username**: the username for the remote machine
            - **Password**: the password for the remote machine
        - **RDP settings**:
            - **Ignore server certificate**: select **Enabled** (Depending on the setup of your RDP Server, it might be required to enable this setting)
            - **Enable wallpaper**: optional
            - **Enable font smoothing**: optional
            - **Enable full window dragging**: optional
        - Advanced settings:
            - **Expressions**: optional, using Python you can define custom [expressions](../property-mappings/expression.mdx).

> 3. Click **Finish**.

Reviewer: Excellent edit

### Create Endpoints for the Provider

Finally, you need to create an endpoint for each remote machine. Endpoints are defined within providers; connections between the remote machine and authentik are enabled through communication between the provider's endpoint and the remote machine.

1. Log in to authentik as an admin, and open the authentik Admin interface.
2. Navigate to **Applications > Providers**.
3. Click the **Edit** button on the RAC provider that you previously created.
4. On the Provider page, under **Endpoints**, click **Create**, and provide the following settings:

    - **Name**: define a name for the endpoint, perhaps include the type of connection (RDP, SSH, VNC)
    - **Protocol**: select the appropriate protocol
    - **Host**: the host name or IP address of the remote machine
    - **Maximum concurrent connections**: select a value or use `-1` to disable the limitation
    - **Property mapping**: select either the property mapping that you previously created, or use one of the default settings
    - **Advance settings**: optional

5. Click **Create**.

### Access the remote machine

> To verify your configuration and access the remote machine, go to the **User interface** of your authentik instance. On the **My applications** page click the **Remote Access** application and authentik will connect you to a secure session on the remote machine, in your web browser.

Reviewer: ```suggestion
To verify your configuration and access the remote machine, go to the **User interface** of your authentik instance. On the **My applications** page click the **Remote Access** application and authentik then connects you to a secure session on the remote machine, in your web browser.
```
